src/commands/run.rs

```diff
@@ -170,6 +176,15 @@ impl RunCommand {
 
         let preopen_sockets = self.compute_preopen_sockets()?;
 
+        // Validate coredump-on-trap argument
+        if let Some(coredump_path) = self.coredump_on_trap.as_ref() {
+            if coredump_path.contains("%") {
+                return Err(anyhow!(
+                    "The coredump-on-trap path does not support patterns yet."
```

Comment:
For context, unix kernels allows to template the coredump path (https://sigquit.wordpress.com/2009/03/13/the-core-pattern/) using variables starting with %. We might want to support them in the future.